docs/ops/activation/Tanh_1.md

```diff
@@ -0,0 +1,52 @@
+## Tanh<a name="Tanh"></a> {#openvino_docs_ops_activation_Tanh_1}
+
+**Versioned name**: *Tanh-1*
+
+**Category**: *Activation function*
+
+**Short description**: Tanh activation function.
+
+**Attributes**:
```

Comment:
To align with other ops in activations:
**Attributes**: *Tanh* operation has no attributes.

Reply:
done